Dockerfile

```diff
@@ -1,10 +1,10 @@
-FROM node:12.5.0
+FROM node:13.14.0
```

Comment:
Updating to v.LTS-1, as we did with Persona

Reply:
I see, we had most new services (at least in argo) on 12 right now, because it was the active LTS version, my suggestion is if we upgrade we move to active LTS version (https://nodejs.org/en/about/releases/,  i.e. 14) 

devs could and probably should use nvm, but if this was a deliberate decision then cool with me we will upgrade later

Reply:
Indeed, it was a conscious decision to allow some "outdated environment" tolerance (hence `LTS - 1`), but I agree it'd be ideal to enforce LTS always. Do you suggest I change it to LTS in this PR?

Reply:
let's get this out and create ticket to upgrade to an LTS, I prefer major version changes to have their own releases to focus on vulnerability fixes

Reply:
Sounds splendid! 👌